recipes/cppcommon/all/CMakeLists.txt

```diff
@@ -0,0 +1,38 @@
+cmake_minimum_required(VERSION 3.13)
```

Comment:
I think this could be lower to 3.8 since it's C++17, I dont see anything that requires higher

Reply:
Lowering it to 3.8 gives this cmake error:

> CMake Error at CMakeLists.txt:14 (target_link_libraries):
  Attempt to add link library "fmt" to target "cppcommon" which is not built
  in this directory.
>
>  This is allowed only when policy CMP0079 is set to NEW.

The line in question is:

https://github.com/conan-io/conan-center-index/blob/b87568ce3922fd7c99eb3a9ae1958049c9b3f031/recipes/cppcommon/all/CMakeLists.txt#L14

Complaining that the wrapper is linking the dependencies to the upstream target...